hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/transport/server/ratis/LocalStream.java

```diff
@@ -18,19 +18,18 @@
 
 package org.apache.hadoop.ozone.container.common.transport.server.ratis;
 
+import org.apache.hadoop.ozone.container.keyvalue.impl.StreamDataChannelBase;
 import org.apache.ratis.statemachine.StateMachine;
 
-import java.io.IOException;
 import java.util.concurrent.CompletableFuture;
-import java.util.concurrent.CompletionException;
 import java.util.concurrent.Executor;
 
 class LocalStream implements StateMachine.DataStream {
-  private final StateMachine.DataChannel dataChannel;
+  private final StreamDataChannelBase dataChannel;
```

Comment:
Use `KeyValueStreamDataChannel` instead.  Then, we can keep `StreamDataChannelBase` package private.